### PR TITLE
genesis: Set initial height to 1

### PR DESCRIPTION
**Pull request by abukosek**
The genesis block should have an initial height of 1, since the value 0 is reserved for the latest block in queries (`go/consensus/api/api.go`, `HeightLatest`).

### CODE REVIEW

File: go/oasis-node/cmd/genesis/genesis.go [hunk 1]
```diff
@@ -157,6 +157,7 @@ func doInitGenesis(cmd *cobra.Command, args []string) {
 		ChainID:   chainID,
 		Time:      time.Now(),
 		HaltEpoch: epochtime.EpochTime(viper.GetUint64(cfgHaltEpoch)),
+		Height:    1,
```
Comment (kostko) [inline go/oasis-node/cmd/genesis/genesis.go:160]: Note that this height is currently unused as the underlying consensus backend (Tendermint) does not yet support starting at heights different from 1.

Comment (abukosek) [reply go/oasis-node/cmd/genesis/genesis.go:160]: It's exported in the genesis document and the `GetStatus` method, which is used by external services (e.g. our Rosetta gateway) to determine the height of the genesis block, so having a mismatching value here breaks other things :)

**Comment by abukosek** [comment]
Good idea, I'll add that.

**Comment by abukosek** [comment]
Done, this is now ready for review :)

**Comment by abukosek** [comment]
After a discussion with Yawning, I'm closing this in favor of returning a hardcoded value of `1` as the genesis height in `GetStatus`.

**Pull request closed without merging**